UML-Diagram/Master-UML.puml

```diff
@@ -27,6 +29,7 @@ package "Library system"{
     Library ..* BorrowManager
     Library ..* BookManager
     Library ..* FineManager
+    Library ..* ReturnContext
```

Comment:
Yes it should be implemented on library but in wich menu, i suggest  review Patron Menus, so can be more exact in which class its going to make a use of ReturnContext, for example PatronBorrowMenu ..* ReturnContext.

Reply:
Yeah, as I mention before I forgot to put the instance in the correct class.